packages/backend/src/open_payments/payment/incoming/service.test.ts

```diff
@@ -78,7 +78,7 @@ describe('Incoming Payment Service', (): void => {
       return {
         pollIncomingPaymentCreatedWebhook: true,
         incomingPaymentCreatedPollFrequency: 1,
-        incomingPaymentCreatedPollTimeout: 100
+        incomingPaymentCreatedPollTimeout: 300
```

Comment:
This test periodically fails in CI. I wanted to give the test a bit more breathing room.